lib/crypto/sr25519/sr25519.go

```diff
@@ -331,14 +337,21 @@ func (k *PublicKey) VerifyDeprecated(msg, sig []byte) (bool, error) {
 	}
 
 	t := sr25519.NewSigningContext(SigningContext, msg)
-	ok := k.key.Verify(s, t)
-	if ok {
+	ok, err := k.key.Verify(s, t)
+	if err != nil {
+		return false, fmt.Errorf("verifying signature for sr25519 signing context: %w", err)
```

Comment:
Should we take this error into account? Or just continue below like when `ok` was false?

Reply:
yeah we should take it into account, cause it might be an encoding error or some other error